sw/airborne/firmwares/rotorcraft/stabilization/wls/wls_alloc.h

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
please add proper header, some docs for the functions, and the includes could probably be moved to the C files